<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MTB-USB konfigurátor</string>
  </property>
  <property name="locale">
   <locale language="Czech" country="Czechia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Připojení k MTB</string>
    </property>
    <widget class="QLineEdit" name="le_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_port">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>3841</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresa</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_connect">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Připojit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_disconnect">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Odpojit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLoadOffline">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Otevřít soubor ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSaveOffline">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Uložit ...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbModule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Modul</string>
    </property>
    <widget class="QSpinBox" name="sbModule">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbReboot">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Restart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Nastavení</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leModuleName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbModuleType">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbModuleChangeType">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Změnit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>jméno</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>typ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbModuleChangeName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Uložit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbModuleChangeAddress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Uložit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLoc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>loc</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbServo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>211</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Servo pozice</string>
    </property>
    <widget class="QPushButton" name="pb_minus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_plus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_plusplusplus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+++</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_minusminusminus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_pos_act">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00000</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_servopos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>120</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_servoposset">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>270</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Pozice</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_plusplus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>++</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_minusminus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_servopos2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_servoposset2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Pozice</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_servo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>servo (1-6)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_set_stop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ukončit nastavení</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbIo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>151</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Vstupy/Výstupy</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>výstup</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>vstup</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbFirmware">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Firmware</string>
    </property>
    <widget class="QPushButton" name="pb_fw_upgrade">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Přehrát FW</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_browsefw">
     <property name="geometry">
      <rect>
       <x>253</x>
       <y>19</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_fw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>mtb-unis-v1.hex</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbModuleList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>211</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Modul</string>
    </property>
    <widget class="QListView" name="lvModuleList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbModuleAdd">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/ico_add</normaloff>:/img/ico_add</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbModuleRemove">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/ico_remove</normaloff>:/img/ico_remove</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbModuleAdd">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pbModuleListRefresh">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Obnovit</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbDaemonReload">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>530</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Načíst nastavení ze souboru</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbAddress">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Adresa</string>
    </property>
    <widget class="QSpinBox" name="sbForceAddress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pbForceAddress">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Vynuť adresu</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbDaemonSave">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Uložit nastavení do souboru</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbModuleList">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Seznam modulů</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
